turbo.json

```diff
@@ -8,12 +8,19 @@
       "dependsOn": ["^build"]
     },
     "lint": {},
+    "clean:build": {
+      "cache": false
+    },
     "build": {
       "dependsOn": ["^build"],
       "outputs": ["lib/**"]
     },
-    "build:website": {
+    "build:docs": {
       "dependsOn": ["^build"],
```

Comment:
shouldn't this depend on it's own build?

Reply:
Not sure what you mean? build:docs doesn't rely on its dependeny's build:docs having run

Reply:
right, but this says that build:docs only wait's on it's dependencies build, not it's own build. Is that what we want?

Reply:
Hmm I assumed it would also wait for its own. Either way, it is what we want. We want the functionless-build-tools to build first.